backend/src/test/resources/application-test.yml

```
@@ -12,4 +12,14 @@ spring:
       api-key: sk-proj-test
 #  threads:
 #    virtual:
-#      enabled: true
\ No newline at end of file
+#      enabled: true
+
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (3 > 2)

(empty-lines)